lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import NxGlobalSidebarFooter, { NxGlobalSidebarFooterProps as Props }
+  from '../NxGlobalSidebarFooter';
+
+import NxTextLink from '../../NxTextLink/NxTextLink';
+
+describe('NxGlobalSidebarFooter', function() {
+  const minimalProps: Props = {
+        supportText: 'Support for RSC',
+        supportLink: '#supporturl',
+        releaseText: 'React Shared Components',
+        releaseNumber: '3.1.4',
+        productTagLine: 'Powered by PLAID VILLAIN',
+        showCreatedBy: true
```

Comment:
`minimalProps` is intended to be the minimum that you can pass to the component without it crashing or failing to compile.  Many of these are optional props which shouldn't be present here.  For each optional prop there should be tests both with and without that prop in use to verify functionality in both cases.

Reply:
> `minimalProps` is intended to be the minimum that you can pass to the component without it crashing or failing to compile. Many of these are optional props which shouldn't be present here. For each optional prop there should be tests both with and without that prop in use to verify functionality in both cases.

Ah, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/dcb088dbde236bda5ab9f146b439c8b4beb1d501